src/main/java/org/mule/services/http/impl/service/server/grizzly/GrizzlyHttpRequestAdapter.java

```diff
@@ -46,21 +47,19 @@
   private ParameterMap queryParams;
 
   public GrizzlyHttpRequestAdapter(FilterChainContext filterChainContext, HttpContent httpContent) {
-    this.filterChainContext = filterChainContext;
     this.requestPacket = (HttpRequestPacket) httpContent.getHttpHeader();
-    isTransferEncodingChunked = httpContent.getHttpHeader().isChunked();
+    isTransferEncodingChunked = requestPacket.isChunked();
```

Comment:
Are these two really the same? Just want to make sure it wasn't done the other way for some specific reason.